extensions/storage-shares/pkg/config/defaults/defaultconfig.go

```diff
@@ -30,6 +30,7 @@ func DefaultConfig() *config.Config {
 		Reva: &config.Reva{
 			Address: "127.0.0.1:9142",
 		},
+		MountID:                "1284d238-aa92-42ce-bdc4-0b0000009154",
```

Comment:
SharesstorageproviderID is `a0ca6a90-a365-4782-871e-d44447bbc668` see utils.ShareStorageProviderID in reva. This is the UserStorageProviderID afaik.